src/lib/relicAugmenter.js

```diff
@@ -13,12 +13,6 @@ export const RelicAugmenter = {
       return null
     }
 
-    relic.weights = {
-      current: 0,
-      average: 0,
-      best: 0,
-    }
-
```

Comment:
I wanted weights to be optional rather than having to update this file every time we change a field in the 'weights' substructure